powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/internal/SystemWrapper.java

```diff
@@ -0,0 +1,10 @@
+package software.amazon.lambda.powertools.logging.internal;
+
+public class SystemWrapper {
+    public SystemWrapper() {
```

Comment:
This should be private?

Reply:
inspired from ->aws-embedded-metrics...:). purely for unittest.. mockito probably requires public constructor for the instrumentation(not confirmed though)

Reply:
I believe it will work with private constructor and also make the class as default.

Reply:
you are right. Changed.